src/components/CartItem/cartItem.module.css

```diff
@@ -0,0 +1,49 @@
+.container {
+  display: grid;
+  background-color: var(--color-bg);
+  padding: 2.5rem;
+  gap: 2.875rem; /* 46px */
+  max-width: 48.875rem; /* 782px */
```

Comment:
It should not have max width. The page container should has max-width and this should be x/y of container.

Reply:
Fixed